.github/workflows/get_dockerfile_path.py

```diff
@@ -0,0 +1,64 @@
+import os
+import yaml
+
+def extract_organization_name(pull_request_description):
+    try:
+        data = yaml.safe_load(pull_request_description)
+        model_details = data.get('Model Details', {})
+        org_name = model_details.get('Organization Name', None)
+        return org_name
+    except Exception as e:
+        print(f"Error while parsing YAML: {e}")
+        return None
+
+def get_pull_request_description():
+    # Fetch the GitHub Pull Request event payload using GitHub API
+    pr_event_url = os.environ.get('GITHUB_EVENT_PATH', None)
+    if pr_event_url is None:
+        print("Error: GITHUB_EVENT_PATH environment variable not set.")
+        exit()
+
+    with open(pr_event_url, 'r') as file:
+        pr_event_data = yaml.safe_load(file)
+
+    # Extract the pull request description from the payload
+    pull_request_description = pr_event_data.get('pull_request', {}).get('body', None)
+
+    if pull_request_description is None:
+        print("Error: Pull Request Description not found in the event payload.")
+        exit()
+    else:
+        return pull_request_description
+    
+def get_latest_model_name():
+    pull_request_description = get_pull_request_description()
+    
+    org_folder = extract_organization_name(pull_request_description)
+    
+    # Cd into the org folder
+    os.chdir(org_folder)
+
+    # Assign current directory to model_n
+    model_n = os.getcwd()
+
+    if model_n == 'objects':
+        exit()
+    else:
+        return model_n
+    
+def get_dockerfile_path(model_folder):
```

Comment:
Same. I saw this earlier. I see that you are requesting the organization name using a PR template. We should also request the model version number and then go to the dockerfile that way instead of the `os.walk` method. Because as I mentioned earlier, this function will return multiple paths if there are multiple versions of a model in the directory.